Makefile

```diff
@@ -2,6 +2,11 @@
 USER_ID := $(shell id -u)
 GROUP_ID := $(shell id -g)
 IN_DOCKER_IMG := $(shell test -f /.dockerenv && echo 1 || echo 0)
```

Comment:
I was able to run tests on cloud from user space. I suggest adding: 
```
IN_DOCKER_IMG := $(shell ( test -f /.dockerenv && echo 1 ) || ( test -f /opt/impact/notebook_manifest.json && echo 1 ) || echo 0)
```
to enable that out of the box